cmd/gitserver/server/server_test.go

```diff
@@ -504,6 +507,7 @@ func addCommitToRepo(cmd func(string, ...string) string) string {
 
 func makeTestServer(ctx context.Context, repoDir, remote string, db database.DB) *Server {
 	s := &Server{
+		Logger:           logger.Scoped("Server", "a gitserver server"),
```

Comment:
This is a test function I think, so you want to use `logtest` instead, and have `makeTestServer` accept the `testing.T` parameter. For example:

```suggestion
func makeTestServer(ctx context.Context, t *testing.T, repoDir, remote string, db database.DB) *Server {
	s := &Server{
		Logger:           logtest.Scoped(t),
```